cpp/ql/lib/semmle/code/cpp/metrics/MetricClass.qll

```diff
@@ -238,7 +238,7 @@ class MetricClass extends Class {
    * classes should add behavior to their base classes, but not alter that
    * behavior dramatically.
    */
-  float getSpecialisationIndex() {
+  float getSpecializationIndex() {
```

Comment:
Please revert this. There are special steps that need to be followed when renaming a predicate (aliasing the old name, marking that as deprecated, and writing up a change note). We might want to do that, but it's probably better done in a separate PR.